pkg/agent/containerd/containerd.go

```diff
@@ -86,7 +86,7 @@ func Run(ctx context.Context, cfg *config.Node) error {
 		}
 
 		logrus.Infof("Running containerd %s", config.ArgString(args[1:]))
-		cmd := exec.Command(args[0], args[1:]...)
+		cmd := exec.CommandContext(ctx, args[0], args[1:]...)
```

Comment:
nit: 
```suggestion
		cmd := exec.CommandContext(ctx, args...)
```

Reply:
CommandContext takes 3 args

Reply:
no, it takes an unlimited number of string args; that's what the ... means. It's jut a nit either way thogh.